Find your receipt or invoice [HC-1548] (#378)
arduino/help-center-content: content/Store Support/Payments and Invoices/Find-your-receipt-or-invoice.md

---
title: "Find your receipt or invoice"
id: 360016121859
---

> When you make a purchase with us, your invoice or receipt is sent to the email address you provided at checkout.
> 
> ## Find your receipt or invoice
> 
> Look for an email from us in your inbox. This email is sent after your purchase is processed. The sender details differ depending on whether you purchased from our EU or US store.

Reviewer: ```suggestion

Learn how to find a receipt or invoice for your purchase.

In this article:

* [Find the receipt or invoice for an Arduino Store purchase](#arduino-store)
* [Find the receipt or invoice for an Amazon purchase](#amazon)

## Find the receipt or invoice for an Arduino Store purchase {#arduino-store}

When you make a purchase in the [Arduino Store](https://store.arduino.cc/), your invoice or receipt is sent to the email address you provided at checkout.

The sender details differ depending on the region:
```
* The first sentence is good, but doesn't apply to Amazon. Moved it to the Arduino Store section.
* Also updated heading names to more clearly reflect the content of those sections.
* Added a TOC.
* Used the more customer-facing "region" choice instead of US/EU to distinguish between stores.

> **For the EU store:**

Reviewer: ```suggestion
**Europe and Africa:**
```
Used the more customer-facing "region" choice instead of US/EU to distinguish between stores.

- **Sender Name:** Admin mail EU (admin.it@arduino.cc)

- **Sender Email:** system@sent-via.netsuite.com

- **Email Subject:** ARDUINO SRL - Invoice No. [Invoice Number] issued on [Date]

> **For the US store:**

Reviewer: ```suggestion
**America, Asia, and Oceania:**
```
Used the more customer-facing "region" choice instead of US/EU to distinguish between stores.

- **Sender Name:** Support arduino (support@arduino.cc)

- **Sender Email:** system@sent-via.netsuite.com

- **Email Subject:** ARDUINO SRL - Invoice No. [Invoice Number] issued on [Date]

If you can’t find the email, use the search function in your email client with the sender, subject, or company name. Check your spam or junk folder if it’s not in your inbox.

> ## Access your Amazon invoice

Reviewer: ```suggestion
## Find the receipt or invoice for an Amazon purchase{#amazon}
```
